tests/components/zwave/test_init.py

```diff
@@ -83,6 +84,36 @@ def side_effect():
     assert network.options.config_path == 'mock_config_path'
 
 
+async def test_network_key_validation(hass, mock_openzwave):
+    """Test network key validation."""
+    test_values = [
+        ('0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07, 0x08, 0x09, 0x0A, 0x0B, '
+         '0x0C, 0x0D, 0x0E, 0x0F, 0x10'),
+        ('0x01,0x02,0x03,0x04,0x05,0x06,0x07,0x08,0x09,0x0A,0x0B,0x0C,0x0D,'
+         '0x0E,0x0F,0x10'),
+    ]
+    for value in test_values:
+        result = zwave.CONFIG_SCHEMA({'zwave': {'network_key': value}})
+        print(result['zwave']['network_key'])
```

Comment:
Print

Reply:
I should be ashamed of myself. How did that end up in a PR...